src/Common/src/TypeSystem/Ecma/EcmaModule.cs

```diff
@@ -165,6 +167,13 @@ protected override IEntityHandleObject CreateValueFromKey(EntityHandle handle)
             }
         }
 
+        private object ResolveModuleReference(ModuleReferenceHandle handle)
+        {
+            ModuleReference moduleReference = _metadataReader.GetModuleReference(handle);
+            string simpleName = _metadataReader.GetString(moduleReference.Name);
```

Comment:
This should be `name` or `fileName`, not simpleName.

Reply:
Thanks, fixed it.